ui/lib/core/addon/helpers/date-format.js

```diff
@@ -43,7 +43,7 @@ function dateFromString(str) {
   return null;
 }
 
-export function dateFormat([value, style], { withTimeZone = false }) {
+export function dateFormat([value, style = 'MMM d yyyy, h:mm:ss aa'], { withTimeZone = false }) {
```

Comment:
I got tired of looking through the codebase to find the format style we use and noticed how inconsistent we can be. I reached out to design to confirm what the default style should be. As a follow-on PR I'll update the other instances of timestamps (like in the card container)